test/index.ts

```diff
@@ -215,6 +215,20 @@ describe('retry-axios', () => {
     scopes.forEach(s => s.done());
   });
 
+  it('should retry when back online', (done) => {
```

Comment:
It doesn't look like this test actually tests the added code.  It would be rad if you could test the specific browser stuff :)